### ISSUE TITLE
Internal: invalid device function

### ISSUE THREAD

**Issue by oletivenu** [tensorflow/nmt#461]
Hi, 

im getting below error after executing this code

### **code:**

python -m nmt.nmt --src=vi --tgt=en --vocab_prefix=/tmp/nmt_data/vocab  --train_prefix=/tmp/nmt_data/train --dev_prefix=/tmp/nmt_data/tst2012  --test_prefix=/tmp/nmt_data/tst2013 --out_dir=/tmp/nmt_model --num_train_steps=12000 --steps_per_stats=100 --num_layers=2 --num_units=128 --dropout=0.2 --metrics=bleu

**### error below :**

2020-02-19 16:44:41.625734: F .\tensorflow/core/kernels/random_op_gpu.h:227] Non-OK-status: GpuLaunchKernel(FillPhiloxRandomKernelLaunch<Distribution>, num_blocks, block_size, 0, d.stream(), gen, data, size, dist) status: Internal: invalid device function
Fatal Python error: Aborted

tensorflow : 1.15
tensorflow-gpu : 1.15
cuda : 10.2

please help me.

**Comment by BBHao1** [comment]
I have the same problem. Have you solved it？
